test/integration/corechecks/docker/exitcode_test.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package docker
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+)
+
+func init() {
+	registerComposeFile("exitcode.compose")
+}
+
+type exitCodeAssertSucceed struct {
```

Comment:
would be great to document why we use this struct, as other teams might dig into the test if it were to fail